Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -293,5 +309,14 @@ protected void ToggleClippingFeature(Material material, bool keywordOn)
                 }
             }
         }
+
+        private void CheckTranformChange()
+        {
+            if (transform.hasChanged)
```

Comment:
I'm always hesitant to use the hasChanged flag since you are unsure who else is mutating it. But, I can't think of any other way to achieve this optimization.